src/caret_analyze/plot/bokeh/pub_sub_info_interface.py

```diff
@@ -52,6 +52,7 @@ def show(
         self._last_full_legends = full_legends
         self._last_export_path = export_path
 
+        # TODO: self._pub_subs is not defined.
```

Comment:
@hsgwa 
Can you add the following sentence since `self._communication` is not defined in communication_interface.py either?

` TODO: self._communication is not defined.`

Reply:
fixed: [fb8dd8e](https://github.com/tier4/CARET_analyze/pull/131/commits/fb8dd8e7ed5b4d1cda14547acc4e3eaf4cd7796d)